action.yml

```
@@ -0,0 +1,38 @@
+name: 'Unleash Feature Flags'
+description: 'Lets you use the getunleash.io feature flags management solution in GitHub Actions'
+inputs:
+  app-name:
+    description: 'The application name for the GitHub Action as you want it to appear in Unleash metrics'
+    type: 'string'
+    required: true
+  api-key:
+    description: 'The frontend API token to use with Unleash'
+    type: 'string'
+    required: true
+  url:
+    description: 'Url to Unleash proxy or frontend api'
```

It's 2023, let's recommend stuff that's up to date!
```suggestion
    description: 'URL to Unleash Edge or Frontend API'
```

---

Indeed